source/appModules/notepadPlusPlus.py

```diff
@@ -46,7 +46,9 @@ class NppEdit(ScintillaBase.Scintilla):
 	def _get_TextInfo(self):
 		if self.appModule.is64BitProcess:
 			appVerMajor, appVerMinor, *__ = self.appModule.productVersion.split(".")
-			if int(appVerMajor) >= 8 and int(appVerMinor) >= 3:
+			# appVerMinor could be either one digit (e.g. in 8.3), two digits (e.g. in 8.21), or even three (8.1.9.2)
+			appVerMinor = appVerMinor.ljust(3, '0')
+			if int(appVerMajor) >= 8 and int(appVerMinor) >= 300:

```

Comment:
Rather than appending zeroes to the minor version you could  just do:
```suggestion
			if int(appVerMajor) >= 8 and int(appVerMinor[0]) >= 3:
```
I won't insist on this though as it is more a matter of personal style than anything else.